### PR TITLE
boards: unified doxygen group names

### PR DESCRIPTION
**Pull request by haukepetersen**
unified the naming style for doxygen group names as decided in #2422

### CODE REVIEW

File: boards/nucleo-l1/include/periph_conf.h [hunk 1]
```diff
@@ -10,7 +10,7 @@
  * @ingroup     board_nucleo-l1
```
Comment (authmillenon) [inline boards/nucleo-l1/include/periph_conf.h]: `s/board_/boards_/`

**Comment by authmillenon** [comment]
Apart from one missing rename: ACK

File: boards/nucleo-l1/include/periph_conf.h (hunk 1 above)
Comment (haukepetersen) [inline boards/nucleo-l1/include/periph_conf.h]: dammit :-)

**Comment by haukepetersen** [comment]
fixed l1 and squashed. Waiting for Travis to make sure I didn't miss anything

**Comment by haukepetersen** [comment]
all seems good. And go.

**Pull request merged**